src/slurm_plugin/computemgtd.py

```diff
@@ -26,6 +26,7 @@
 from slurm_plugin.common import CONFIG_FILE_DIR, TIMESTAMP_FORMAT, InstanceManager, log_exception, time_is_up
 
 LOOP_TIME = 60
+RELOAD_CONFIG_ITERATIONS = 10
```

Comment:
Maybe make this smaller so that time between reloading is small than clustermgtd_timeout? Otherwise it's impossible to change clustermgtd_timeout and "save" a node

Reply:
For now I think it's ok. New nodes will pull the new config.